fuel-asm/src/panic_instruction.rs

```diff
@@ -35,6 +37,35 @@ impl PanicInstruction {
     }
 }
 
+/// Helper struct to debug-format a `RawInstruction` in `PanicInstruction::fmt`.
+struct InstructionDbg(RawInstruction);
+impl fmt::Debug for InstructionDbg {
+    /// Formats like this: `MOVI { dst: 32, val: 32 } (bytes 72 80 00 20)`}`
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match Instruction::try_from(self.0) {
+            Ok(instr) => write!(f, "{:?}", instr)?,
+            Err(_) => write!(f, "Unknown")?,
```

Comment:
Nit: Maybe something like `Unknown Instruction` or `Unknown Operation' might be slightly more clear when the caller doesn't know what to expect in this field. I don't feel strongly about it though.

Reply:
It already says `instruction: Unknown`, so I think it's good enough